bin/changelog/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const inquirer = require( 'inquirer' );
+const chalk = require( 'chalk' );
+const { REPO } = require( './config' );
+const { make } = require( './make' );
+
+/* eslint no-console: 0 */
+const success = chalk.bold.green;
+
+const makeChangeLog = async () => {
+	console.log(
+		chalk.bold( '💃 Time to prepare the Gutenberg Changelog 🕺\n\n' )
+	);
+
+	const { token } = await inquirer.prompt( [
+		{
+			type: 'input',
+			name: 'token',
+			message:
+				'Please provide a GitHub personal authentication token. Navigate to ' +
+				success(
+					'https://github.com/settings/tokens/new?scopes=public_repo,admin:org,user'
```

Comment:
I haven't looked in much detail for the specifics of the requests, but if it's only retrieving publicly-available information about milestones and pull requests, it shouldn't need any scope privileges. It's better to require fewer scopes whenever possible, since each poses a potential risk if the token was ever leaked.

Reply:
It may not even be necessary to have a token at all. Anonymous requests are _very_ limited in rate limiting, but if we can optimize requests to occur in fewer than would likely be encountered by rate limiting, it would certainly improve the overall experience a fair amount. Sometimes it can become an issue when running the script multiple times (especially true during initial development or maintenance). While it'd be more work, a nice compromise could be: Try anonymous requests up until the point that the rate limit is reached, and only then prompt for a token to continue the requests.

Reply:
> It may not even be necessary to have a token at all.

Apparently the GraphQL endpoint requires authentication for all requests, which is not the case for the REST API.